st2client/st2client/utils/strutil.py

```diff
@@ -15,5 +15,5 @@
 
 
 def unescape(s):
-    return (s.replace('\\r\\n', '\\n').decode('unicode_escape').encode('utf-8')
+    return (s.replace('\r\n', '\n')
```

Comment:
No need to escape `\`. Python treats \r as a single character.

In [1]: a = '\r'

In [2]: len(a)
Out[2]: 1

In [3]: